spec/mobility/plugins/active_record/query_spec.rb

```diff
@@ -192,4 +192,29 @@
       expect { query.order(:car_id) }.not_to raise_error
     end
   end
+
+  describe "regression for #564" do
+    it "works if translates is called multiple times" do
+      stub_const 'Article', Class.new(ActiveRecord::Base)
+      2.times { translates Article, :title, backend: :table }
+
+      article = Article.create(title: "Title")
+
+      expect(Article.i18n.where(title: "Title")).to eq([article])
+    end
+
+    it "handles intersecting attribute declarations" do
+      stub_const 'Article', Class.new(ActiveRecord::Base)
+      translates Article, :title, :content, backend: :key_value, type: :string
+
+      # title defined below clobbers title defined above
+      translates Article, :title, backend: :table
```

Comment:
Maybe we should issue a warning, but for now this at least handles the situation gracefully.